portfolio/src/main/webapp/script.js

```diff
@@ -48,7 +48,13 @@ function fetchAndAddComments(){
  */
 function createCommentElement(comment) { 
   const commentElement = document.createElement('li');
-  commentElement.appendChild(document.createTextNode(comment['username']+ "\n"));
+  commentElement.appendChild(document.createTextNode(comment['username']+ " - "+ "\n"));
```

Comment:
weird spacing: comment['username]+ "-"+ "\n" should be:

comment['username] + "-" + "\n"